docs/blog/oras-looking-back-at-2022-and-forward-to-2023.md

```diff
@@ -0,0 +1,88 @@
+- These contributors come from [35 companies](https://oras.devstats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20year&var-metric=contributions)
+- All [Stars](https://oras.devstats.cncf.io/d/3/stars-and-forks-by-repository?orgId=1&from=now-1y&to=now) increased from 605 to 964, all [Forks](https://oras.devstats.cncf.io/d/3/stars-and-forks-by-repository?orgId=1&from=now-1y&to=now) increased from 123 to 220 in the past year
+- The total downloads of ORAS CLI are 600,368
+- [The number of new PRs has tripled in the last year](https://oras.devstats.cncf.io/d/15/new-prs-in-repository-groups?orgId=1)
+- Organized 17 public community meetings in 2022, see [meeting notes](https://hackmd.io/P-O6n222TcSMoJgHmTTduw?view)
+
+## Adoption: Powering multiple industries and OSS communities
+
+ORAS CLI, ORAS Go, and Python SDK are designed to help users and developers manage [OCI Distribution](https://github.com/opencontainers/distribution-spec) based artifacts. ORAS empowers the secure supply chain by enabling users to leverage the existing services they already have across their development to production environments.
+
+Currently, the biggest cloud providers like Microsoft Azure, AWS, and Google Cloud are using ORAS to manage OCI artifacts in registries. ORAS Go SDK has been integrated and adopted by some industry-leading vendors and popular open-source projects. Here is part of known adopters till now:
+
+- [Amazon ECR](https://aws.amazon.com/ecr/)
+- [Amazon EKS Anywhere](https://anywhere.eks.amazonaws.com/docs/workshops/packages/harbor/#set-up-trivy-image-scanner-in-an-air-gapped-environment)
+- [Alibaba Cloud Service Mesh](https://www.alibabacloud.com/help/en/alibaba-cloud-service-mesh/latest/use-oras-to-simplify-wasm-based-asm-instance-extension)
+- [Artifact Hub](https://artifacthub.io/docs/topics/repositories/helm-charts/#oci-support)
+- [Docker Hub](https://www.docker.com/blog/announcing-docker-hub-oci-artifacts-support/)
+- [GitHub](https://github.com/)
+- [Google Cloud](https://cloud.google.com/anthos-config-management/docs/how-to/sync-oci-artifacts-from-artifact-registry#oras)
+- [Helm](https://v3.helm.sh/docs/topics/registries/)
+- [Singularity](https://sylabs.io/guides/3.1/user-guide/cli/singularity_push.html)
+- [Microsoft Azure - ACR](https://azure.microsoft.com/en-us/products/container-registry/)
+- [Notary v2](https://github.com/notaryproject/notation)
+- [KubeApps](https://github.com/vmware-tanzu/kubeapps) by VMware Tanzu
+- [VMware Application Catalog](https://tanzu.vmware.com/application-catalog)
+- [UOR Framework](https://universalreference.io/) by Red Hat
```

Comment:
Thank you for the shout-out! We've renamed UOR Framework to Emporous recently. Might you please change this entry for UOR Framework to "Emporous (Formerly UOR Framework)"?

Reply:
Renamed to "Emporous (Formerly UOR Framework)"